packages/components/card/src/card/__snapshots__/Card.test.tsx.snap

```diff
@@ -0,0 +1,232 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`can be selected 1`] = `
```

Comment:
We decided to delete snapshot tests, so this file should be gone, together with a corresponding test.